benchmark/common.js

```diff
@@ -1,6 +1,7 @@
 'use strict';
 
 const child_process = require('child_process');
+const http_benchmarkers = require('./http-benchmarkers.js');
```

Comment:
perhaps it should be called `_http-benchmarkers.js`, that is how the other utility files are named.